Gems/AtomLyIntegration/AtomViewportDisplayInfo/Code/Source/AtomViewportDisplayInfoSystemComponent.cpp

```diff
@@ -288,7 +303,7 @@ namespace AZ::Render
             AZStd::string::format(
                 "FPS %.1f [%.0f..%.0f], %.1fms/frame, avg over %.1fs",
                 averageFPS,
-                minFPS,
+                minFPS == DBL_MAX ? 0.0 : minFPS,
```

Comment:
Will this fix the super annoying FPS print that fills up the whole screen in the Editor!? 😄

Reply:
Yes it will :)